python/caffe/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .pycaffe import Net, SGDSolver, NesterovSolver, AdaGradSolver, RMSPropSolver, AdaDeltaSolver, AdamSolver
 from ._caffe import set_mode_cpu, set_mode_gpu, set_device, Layer, get_solver, layer_type_list
+from ._caffe import CAFFE_VERSION as __version__
```

Comment:
Drop in favor of exporting as `__version__` on the boost::python side unless there's an obstacle there.